sdcm/cluster.py

```diff
@@ -3050,11 +3053,12 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
         # NOTE: following is needed in case of K8S where we init multiple DB clusters first
         #       and only then we add nodes to it calling code in parallel.
         if add_nodes:
+            run_add_nodes = self.add_db_nodes if 'db' in self.node_type else self.add_nodes
             if isinstance(n_nodes, list):
                 for dc_idx, num in enumerate(n_nodes):
-                    self.add_nodes(num, dc_idx=dc_idx, enable_auto_bootstrap=self.auto_bootstrap)
+                    run_add_nodes(num, dc_idx=dc_idx, enable_auto_bootstrap=self.auto_bootstrap)
```

Comment:
Is there a problem with leaving current `self.add_nodes(n_nodes, enable_auto_bootstrap=self.auto_bootstrap)` and later loop over added nodes and trigger collect kallsyms?
Current solution is hiding actions. Names don't show the reason we wrapped add_nodes function.
e.g. solution:
```
new_nodes = self.add_nodes(num, dc_idx=dc_idx, enable_auto_bootstrap=self.auto_bootstrap)
self._trigger_collect_kallsyms_map_for_db_nodes(new_nodes)  # as we do it in parallel - we just trigger collection
```

Reply:
@soyacz the point to notice here is that according to your suggestion, all nemeses (or other code) that runs `add_nodes` will also have to duplicate the call of `self._trigger_collect_kallsyms_map_for_db_nodes(new_nodes)` and you probably don't want this risk.

Reply:
so when nemesis adds node then we don't run it?

Reply:
@soyacz , right, instead of having multiple duplicated calls for it on each nemesis, we simply have a single call by introducing this `run_add_nodes` calling `add_db_nodes` wrapper.